README-QUIC.md

```diff
@@ -0,0 +1,97 @@
+Using OpenSSL QUIC
+==================
+
+From OpenSSL 3.2, OpenSSL features support for client-side QUIC.
+
+Users interested in using the new QUIC functionality are encouraged to look at
+some of the following resources:
+
+- The [openssl-quic(7) manpage], which provides a basic reference overview of
+  QUIC functionality and how use of QUIC differs from use of TLS with regard to
```

Comment:
`... how creating QUIC connections differs from creating TLS connections with...`

Reply:
Not really sure about this one, it is a lot broader than just how connections are made.

Reply:
OK